### ISSUE TITLE
Get-SFDrive: DriveID and Status mutually exclusive

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#31]
Not sure if this is a feature or bug:

```
PS C:\Program Files\SolidFire> Get-SFDrive -DriveID 2 -Status active
Get-SFDrive : Parameter set cannot be resolved using the specified named parameters.
At line:1 char:1
+ Get-SFDrive -DriveID 2 -Status active
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-SFDrive], ParameterBindingException
    + FullyQualifiedErrorId : AmbiguousParameterSet,SolidFire.Drive.Get.GetSFDrive
```

`Get-SFDrive -DriveID 2` works and  `Get-SFDrive -Status active` lists DriveID 2, but it's not possible to match the both in one go. 

Without this I need to use one of these commands and then check for the other condition which isn't too bad. Online help clearly says `-DriveID` only queries the specified drive, but it doesn't say that `-Status` cannot be used with it (which does make sense, but if Status wasn't matched it'd also make sense if `Get-SFDrive -DriveID 2 -Status active` worked, but returned nothing.)

And I wonder if this error message (Parameter set cannot be resolved using the specified named parameters) could be improved - the parameters are correct, although their combined use may not be supported.

**Comment by ahaid** [comment]
This is a good catch. I created ticket SDK-873 in the next sprint to address this before 1.5 is released.

**Comment by scaleoutsean** [comment]
Thanks, Adam. 
Are issues like this common across languages (that is, would it make sense to recheck this in solidfire-cli and/or submit the same bug there)? If an issue appears in both languages, does a fix also result in tests that apply both to SolidFire PS Tools and the CLI?

**Comment by ahaid** [comment]
This problem only applies to the PowerShell module. The SDKs and API use all filter criteria present in their requests.

**Comment by ahaid** [comment]
This issue was fixed in the 1.5 release.
